challenge.md

```diff
@@ -6,19 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+After you fork a repo you copy the SSH URL and then type 'git clone [SSH URL]' in the terminal
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git diff HEAD^x HEAD^y
```

Comment:
We were looking for `git status`, but given how the question was worded, `git diff` is a reasonable answer - I won't take off any points. In the real world, you'll be using both `git status` *and* `git diff` to review and inventory the changes you've made.